monix-reactive/shared/src/main/scala/monix/reactive/Observable.scala

```diff
@@ -739,7 +739,47 @@ abstract class Observable[+A] extends Serializable { self =>
     */
   final def bufferTimedWithPressure(period: FiniteDuration, maxSize: Int): Observable[Seq[A]] = {
```

Comment:
I think we could leave one method with default `sizeOf` argument. It should also fix doc ambiguity issue

Reply:
Ok I'm going to change that. Thank you.